crm_claim/models/crm_claim.py

```diff
@@ -6,6 +6,7 @@
 import odoo
```

Comment:
The linter tells you to remove this now: https://travis-ci.org/OCA/crm/jobs/300289352#L738